tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
[curious] It might be worthwhile to consider how we might be able to define a standard `tsconfig` in `@edx/frontend-build` such that:
* It may be shared with consuming applications so they don't need to re-define their own in the root of their repos, too.
* It may be extended/customized by consuming applications.

Reply:
See https://github.com/sindresorhus/tsconfig as an example repo that hosts a shared TypeScript config that gets extended by consuming applications:

```json
{
	"extends": "@sindresorhus/tsconfig",
	"compilerOptions": {
		"outDir": "dist"
	}
}
```

Reply:
I looked around for mechanisms that we might be able to export the tsconfig similar to how we export things like the config.js's, but there don't appear to be any built-in mechanisms yet (much to the chagrin of many developers).  The closest thing I could find was an external library https://github.com/c-vetter/tsconfig.js

Reply:
Started conversation on creating that repo: https://github.com/openedx/tcril-engineering/issues/643